JSQMessagesViewController/Views/JSQMessagesTypingIndicatorFooterView.m

```diff
@@ -72,9 +71,18 @@ - (void)setBackgroundColor:(UIColor *)backgroundColor
 }
 
 #pragma mark - Typing indicator
+// For backward compability
```

Comment:
this will be part of the 8.0 release, so we don't need to worry about backward compatibility 😄